pynetsnmp/netsnmp.py

```diff
@@ -406,15 +442,32 @@ def netsnmp_logger(a, b, msg):
     ("data", c_void_p),
     ("data_length", c_int),
     ("msgMaxSize", c_size_t),
+    ("base_transport", POINTER(netsnmp_transport)),
     ("f_recv", c_void_p),
     ("f_send", c_void_p),
     ("f_close", c_void_p),
+    ("f_open", c_void_p),
     ("f_accept", c_void_p),
     ("f_fmtaddr", c_void_p),
+    ("f_config", c_void_p),
+    ("f_copy", c_void_p),
+    ("f_setup_session", c_void_p),
+    ("identifier", POINTER(u_char_p)),
+    ("f_get_taddr", c_void_p),
```

Comment:
Can you make these changes conditional based on the version of the net-snmp library similarly to how the changes to the `netsnmp_session` structure were made?

Reply:
I'll do that, but it seems they have existed even in the 5.7.2 version based on the source code of net-snmp, frankly, I'd would rather clean the code out of unnecessary versioning

Reply:
If these fields existed in the 5.7.2 version, then the prior code in pynetsnmp was disastrously wrong.